app/views/layouts/application.html.erb

```diff
@@ -5,12 +5,12 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
   </head>
-
   <body>
-    <%= yield %>
-  </body>
+<%= render "shared/navigation"%>
```

Comment:
Check the indentation here

Reply:
@YilKanda we already corrected it, thanks :smile: